SecurityScorecard/vulnerability_assessment_scanner/_meta/fields.yml

```diff
@@ -0,0 +1,19 @@
+scorecard.id:
```

Comment:
I'm wondering if the name should be like this `scorecard.vas.id`, as there's other services.